### PR TITLE
PolyhedralGeometry: Format most of the files

### PR DESCRIPTION
**Pull request by lkastner**
I am only aware of #3512 and excluded the two files being worked on there.

### CODE REVIEW

**Comment by lgoettgens** [comment]
Thanks, @lkastner, for taking care of this!

Wouldn't it be nicer for the history if the new action and formatting of the PolyhedralGeometry folder were in different PRs? If you need some formatted files to test the action with, there is already the LieAlgebras stuff you found.

File: etc/test_formatting.jl [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          "experimental/LieAlgebras/src/LieSubalgebra.jl",
+          "experimental/LieAlgebras/src/LinearLieAlgebra.jl",
+          "experimental/LieAlgebras/src/RootSystem.jl",
+          "experimental/LieAlgebras/src/Util.jl",
+          "experimental/BasisLieHighestWeight/src/MainAlgorithm.jl",
+         ]
+
+  # Collect all code files.
+  entire = [
+            _gather_source_files(joinpath(oscardir, "src/"));
```
Comment (lgoettgens) [inline etc/test_formatting.jl:67]:
```suggestion
            _gather_source_files(joinpath(oscardir, "src/")); 
            _gather_source_files(joinpath(oscardir, "test/")); 
```

**Comment by benlorenz** [comment]
> Wouldn't it be nicer for the history if the new action and formatting of the PolyhedralGeometry folder were in different PRs? If you need some formatted files to test the action with, there is already the LieAlgebras stuff you found.

If we make sure not to squash this PR then the commits stay separate, also with a proper merge instead of a rebase to make sure the commit for the blame-ignore-rev stays correct.

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Ran JuliaFormatter on most of src/PolyhedralGeometry
+ab444bbc46b5493588b2c3217ed35d86396d1260
```
Comment (lgoettgens) [inline .git-blame-ignore-revs:2]: This is now wrong due to the rebase

**Review by fingolfin — APPROVED** [review]

**Pull request merged**